internal/core/cmake/Utils.cmake

```diff
@@ -72,6 +72,21 @@ MACRO(get_milvus_version)
     endif()
 ENDMACRO(get_milvus_version)
 
+# get knowhere version
+MACRO(get_knowhere_version)
```

Comment:
due to duplicated file name with knowhere project, get_knowhere_version command cann't be found.

Reply:
this is a knohwere bug

Reply:
I see. Any fix suggestion? Renaming either one of the cmake file?

Reply:
I will rename this file of knowhere to fix it.

Reply:
Please revert get_knowhere_version